CHANGELOG.md

```diff
@@ -1,6 +1,39 @@
 Change Log
 ==========
 
+Version 3.3.0
+-------------
+
+_2020-09-12_
+
+ * New: Proto3 support! This includes the new behaviors, the new types, and the JSON. Only for JVM
```

Comment:
Curious why Proto3 is JVM only, is that because of the duration types?

Reply:
I don't think proto3 has been addressed for Swift. I don't believe there are tests, JSON support, adequate equivalent generated type for the new protobuf things, etc. As far as I know anyway.

Reply:
Swift has lots of proto3 support. That aside, what about JS and native? Those should also have support already.

Reply:
FWIW I converted my demo to proto3 and it works flawlessly: https://github.com/Egorand/wire-multiplatform-sample/commit/872d9eee599f889005c0df51738a6a77322abb7e